src/components/advancedCollapsible/AdvancedCollapsible.tsx

```diff
@@ -18,16 +18,18 @@ export interface AdvancedCollapsibleProps extends Omit<BoxProps, 'size'> {
   children: ReactNode;
   title: string;
   size?: AllowedAdvancedCollapsibleSize;
+  isCollapsed?: boolean;
```

Comment:
NTH: Call this `defaultIsCollapsed` to indicate that changing this prop after the initial render won't change the state anymore

(or alternatively make it so that changing this prop also changes the state)